snapcraft/hooks/disconnect-plug-ovn-run

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+if [ "${openvswitch_builtin:-"false"}" = "true" ]; then
+    export OVS_RUNDIR="${SNAP_COMMON}/openvswitch/run/"
+    (
+        set -e
+        export OVS_LOGDIR="${SNAP_COMMON}/openvswitch/logs/"
+        export OVS_DBDIR="${SNAP_COMMON}/openvswitch/db/"
+        export OVS_SYSCONFDIR="${SNAP_COMMON}/openvswitch/conf/"
+        export OVS_PKGDATADIR="${SNAP}/share/openvswitch/"
+        export OVS_BINDIR="${SNAP}/bin/"
+        export OVS_SBINDIR="${SNAP}/bin/"
+
+        mkdir -p "${OVS_SYSCONFDIR}/openvswitch"
```

Comment:
Not sure if there should be some cleanup here in case we set up `openvswitch_builtin`in `daemon.start`, then installed and uninstalled MicroOVN.